projects/distributed-tracing/src/shared/dashboard/widgets/table/mode-toggle-table-widget.model.ts

```diff
@@ -84,4 +85,9 @@ export class ModeToggleTableWidgetModel extends TableWidgetBaseModel {
   public getChildModel(row: TableRow): object | undefined {
     return this.delegateModel && this.delegateModel?.getChildModel(row);
   }
+
+  public getRowSelectionHandlers(row: TableRow): TableWidgetRowSelectionModel[] {
+    return (this.delegateModel && this.delegateModel?.getRowSelectionHandlers(row).length > 0) ?
```

Comment:
Is it same as `this.delegateModel?.getRowSelectionHandlers(row) ?? []` ?

Reply:
Not quite since that could return undefined.

Reply:
if the first part returns undefined, then it would return [] to the caller